go.mod

```diff
@@ -23,3 +24,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+replace github.com/lightninglabs/gozmq => github.com/wpaulino/gozmq v0.0.0-20190709225702-f5caaa46981f
```

Comment:
Can be removed now in favor of updating the dep directly.

Reply:
Done.